.github/workflows/main.yml

```
@@ -0,0 +1,80 @@
+        git -C ./ status
+        git -C ./ add owa_env.php
+        git -C ./ add package.json
+        git -C ./ commit -m "Updating version string."
+        echo Pushing change into Branch
+        git -C ./ push --set-upstream origin release-${{ env.VERSION }}
+    # install the composer vendor directory
+    - name: Install Composer Dependancies
+      uses: php-actions/composer@v1
+    # setup node and build js/asset bundles
+    - name: Setup Node/npm 
+      uses: actions/setup-node@v2
```

actionlint (1.7.7)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)